generate/generate.go

```diff
@@ -895,13 +895,23 @@ func (g *Generator) qualifiedTypeName(typeName string, pkg *types.Package) strin
 func (g *Generator) convertType(typ types.Type) (desc.FieldDescriptorProto_Type, desc.FieldDescriptorProto_Label, string) {
 	switch typ := typ.(type) {
 	case *types.Basic:
-		// Map Go types to proto types via
+		// Map Go types to proto types
```

Comment:
If you want to split these two lines into two sentences, add some punctuation, like `:`.

Reply:
Done.